### PR TITLE
CORE-1464 - Vault alias decorator

### PR DESCRIPTION
**Pull request by k-sever**
## Description of changes in release / Impact of release:
Add support of alias decorator in larky vault module

## Documentation
(insert text here)

### Is this a breaking change?
- [ ] Yes
- [X] No

### If you answered Yes then describe why is it so
(insert text here if applicable)

### Is there a way to disable the change?
- [X] Use previous release
- [ ] Use a feature flag
- [ ] No

#### Additional details go here
(insert text here if applicable)

### CODE REVIEW

File: larky/pom.xml [hunk 1]
```diff
@@ -43,6 +43,7 @@
         <google.jimfs.version>1.2</google.jimfs.version>
         <google.re2j.version>1.6</google.re2j.version>
         <google.truth.version>1.1.3</google.truth.version>
+        <google.gson.version>2.9.1</google.gson.version>
```
Comment (mahmoudimus) [inline larky/pom.xml:46]: Why do you need this? We already have a json parser that we've built as part of internal starlark.

File: larky/src/main/java/com/verygood/security/larky/modules/vgs/vault/spi/LarkyVault.java [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 public interface LarkyVault extends StarlarkValue {
 
-    Object redact(Object value, Object storage, Object format, List<Object> tags) throws EvalException;
+    Object redact(Object value, Object storage, Object format, List<Object> tags, Object decoratorConfig) throws EvalException;
```
Comment (mahmoudimus) [inline larky/src/main/java/com/verygood/security/larky/modules/vgs/vault/spi/LarkyVault.java:10]: Should we move this to a builder pattern instead? Or do we just want to continue expanding the parameters? Does "decoratorConfig" really belong as a parameter ?

File: larky/pom.xml (hunk 1 above)
Comment (mahmoudimus) [reply larky/pom.xml:46]: I am not OK w/ adding another dependency tbh.

File: larky/src/main/java/com/verygood/security/larky/modules/vgs/vault/spi/LarkyVault.java (hunk 2 above)
Comment (Agitolyev) [reply larky/src/main/java/com/verygood/security/larky/modules/vgs/vault/spi/LarkyVault.java:10]: I believe it makes API simpler to have a decorator as a parameter.
1. Builder adds some notion of state inside a script, complicating things.
2. You don't need to instantiate different objects if you have to tokenize different PII entities using different approaches:
``` python
load('@stdlib/json', 'json')
load('@stdlib//builtins', 'builtins')
load('@vgs//vault', 'vault')

def process(input, ctx):
    json_body = json.loads(str(input))
    
    json_body['card_number'] = vault.redact(json_body['card_number'], decoratorConfig=...)
    json_body['ssn'] = vault.redact(json_body['ssn'])
    return json.dumps(json_body)
```
3. Explicit is better than implicit.

File: larky/pom.xml (hunk 1 above)
Comment (k-sever) [reply larky/pom.xml:46]: agree, I didn't see we have one. Will use the internal one, thx

**Review by nestorsokil — APPROVED** [review]

**Review by Agitolyev — APPROVED** [review]

**Pull request merged**